test/data/delta-valid/meta-and-values.json

```diff
@@ -0,0 +1,19 @@
+{
+  "context": "bar",
```

Comment:
Could we have the test files populated with data which looks real ie. context, label, path, etc with sensible values in them

Reply:
see PR #493 for modified tests